### ISSUE TITLE
Being able to NOT send large emoji

### ISSUE THREAD

**Issue by eldelacajita** [telegramdesktop/tdesktop#6846]
### Is your feature request related to a problem?
When I disable "large emoji" in the settings, I stop seeing them, but other people in the chat still see them. So I think I'm sending a small emoji, and I'm actually still sending a huge animated emoji instead, which I don't like and doesn't suit my communication style.

### Describe the solution you'd like
When one user disables "large emoji", the single emoji sent by him/her should remain small like the rest of the emoji he/she is seeing on the chat. 

The content sent by a user should always be WYSIWYG. As a principle, the sender should be able to decide how the message should be shaped. On top of that, the receiver can disable certain features, as it is now.

### Describe alternatives you've considered
- Divide the "large emoji" setting into two: "Show large emoji" and "Send large emoji".
- Alternatively, keep a single setting and when "large emoji" is disabled, also stop sending them.

### Additional context
TL;DR: Large emoji are one of the first features of Telegram I don't like. Being able to hide large emoji is awesome, but sending them without even knowing is quite bad experience. I would love to NOT send them at all.

Edit: Typo.

**Comment by 23rd** [comment]
For the server it is exactly the same plain text as before and server can not tell clients how they should display emoji.
So this issue does not belong to the Telegram Desktop repo.
https://twitter.com/telegram
